cpp/oneapi/dal/algo/louvain/common.cpp

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+* Copyright 2021 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "oneapi/dal/algo/louvain/common.hpp"
+#include "oneapi/dal/detail/error_messages.hpp"
+
+namespace oneapi::dal::preview::louvain::detail {
+
+template <typename Task>
+class descriptor_impl : public base {
+public:
+    double accuracy_threshold = 0.0001;
```

Comment:
why these values are the default?

Reply:
accuracy_threshold = 0.0001 is used in the neo4j
resolution = 1 - original definition of modularity (it is equivalent to the modularity formula without the resolution parameter).